localization/i18n/cs/OrcaSlicer_cs.po

```diff
@@ -3768,7 +3768,7 @@ msgid "Heatbed preheating"
 msgstr "Předehřev vyhřívané podložky"
 
 msgid "Sweeping XY mech mode"
-msgstr "Sweeping XY mech mode"
+msgstr ""
```

Comment:
Will this be ignored or make the text actually be translated into an empty string?